build.psm1

```diff
@@ -1036,6 +1036,53 @@ function Get-PesterTag {
     $o
 }
 
+# Function to build and publish the Microsoft.PowerShell.NamedPipeConnection module for
+# testing PowerShell remote custom connections.
+function Publish-CustomConnectionTestModule
+{
+    # Build test module and run custom connection tests only for Windows platforms
+    if (! $environment.IsWindows) {
```

Comment:
why is this limited to just Windows?

Reply:
I felt it was not needed on all platforms since it only verifies that the newly public types are available, no new functionality is provided or tested.  This reduces the stress of maintaining building the tool and running tests on multiple platforms.

Reply:
If that's the only reason, I think we should enable it for non-Windows in case a future change is platform specific either deliberately or accidentally

Reply:
Ok, will do.